### DISCUSSION TITLE
Gotek Jumper settings?

### DISCUSSION THREAD

**Discussion by debmint** [keirf/flashfloppy#782] [General]
I've just purchased a Gotek and am a little confused in what they all do.  I got it from dasherdeals fon Ebay, and am thriolled with their service.  It is one of the factory modded units and it came from dasherdeals already flashed.  My computer is an OSK system - a PT68K-5 board built by Peripheral Technologies.  It is designed to accept XT wxpansipon cards and uses twisted floppy cable, so I'd assume the Gotek would be jumpered like a PC.

This may be the latest unit.  It came with S1 jumpered, although this pin was not labelled, but I believe it is still S1.  Also, there was a jumper between JA and J5.  I've seen pictures where there is a setup like this.  I contacted dasherdeals and someone calling himself "JJ" replied that I ought to remove the JA-J5 jumper and perhaps place it across JC  which I did.  The drive seems to work well with this setup, but I wonder if I might be doing harm to it with this setup in any way (JC jumpered across and S1(?) jumpered across without JC-J5).

Does anyone know what this JC-J5 jumpering does?  Actually, exactly what does JC do?  I've done quite a bit of searching and definitions for the jumpering is sparse.  I found a diagram showing the function of some of then pins, but for the life of me, I cannot find it again.

Thanks for any help and for such a fantastic product as flashfloppy

**Comment by keirf** [comment]
The JA-J5 jumper is actually across two ground pins and so has no effect. It's just a handy place to store a spare jumper.

JC sets the interface mode to IBM PC compatible. It just changes the behaviour of pins 2 & 34 at the floppy header. Many need this; and equally many don't.

**Reply by debmint** [reply]
> The JA-J5 jumper is actually across two ground pins and so has no effect. It's just a handy place to store a spare jumper.
> 
> JC sets the interface mode to IBM PC compatible. It just changes the behaviour of pins 2 & 34 at the floppy header. Many need this; and equally many don't.

Many thanks for the consideration.  I checked the box that I had researched and had, made several searches but after making the post, went back and checked again and found a thread where someone gave the same answer about JA-J5 as you did.  Someone disputed his, but I came away believing him - and now I do.  As for JC, if it works,there's no harm in keeping it jumpered.  I believe I saw somewhere on the wiki here that you can either jumper this or set host=ibmpc in FF.CFG.  (I believe it's host) and it is also stated that OS9 sets this parameter to the same.

Thanks again and keep up the good work.  These Goteks are fantastic, especially with FlashFloppy.
